src/Components/Landing/Landing.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import "./landing.css";
+
+const Landing = () => {
+  return (
+    <section id="landing">
+      <div className="container">
+        <div className="heading">
+          <h1 className="title">Listening is everything.</h1>
+        </div>
+        <div className="para">
+          Listen to millions of music, without interruptions.
+        </div>
+        <NavLink to={"/auth/signup"} className="signup">
+          Sign up today
```

Comment:
Can we underline this as well as the "Already have an account? Login"? So that it's clear that it's a link

Reply:
Will do.